pkg/replicator/replicator.go

```diff
@@ -111,7 +111,7 @@ func (r *Replicator) newTable(tblName config.PgTableName, tblConfig config.Table
 func (r *Replicator) checkPgSlotAndPub(tx *pgx.Tx) error {
 	var slotExists, pubExists bool
 
-	err := tx.QueryRow("select "+
+	err := (*tx).QueryRow(r.ctx, "select "+
```

Comment:
Lots of changes like this (`tx` => `*tx`) now that go doesn't automatically dereference pointers to interfaces.